docs/source/operator_reference.rst

```diff
@@ -4,23 +4,49 @@
 Operators
 =========
 
-Loading operators.
-
-.. raw:: html
-
-    <iframe
-      src="_static/dpf.html#CPython"
-      style="
-        position: fixed;
-        top: 36px;
-        bottom: 0px;
-        right: 0px;
-        width: 100%;
-        border: none;
-        margin: 0;
-        padding: 0;
-        overflow: hidden;
-        z-index: 1000;
-        height: 100%;
-      ">
-    </iframe>
+DPF operators allow the manipulation and the transformation of simulation data.
+
+From DPF Server for Ansys 2023R2 onwards, the operators available in DPF will depend on the active
+`ServerContext <https://dpf.docs.pyansys.com/api/ansys.dpf.core.server_context.html#servercontext>`_.
+
+The available contexts are **Entry** and **Premium**. Click on the correct section below to discover
+operators available in your context.
+
+.. grid:: 2
+
+   .. grid-item::
+        .. card:: Operators (Entry)
+            :img-top: _static/assets/basic.png
+            :link-type: doc
+            :link: operator_reference_entry
+
+            Click here to get the list of operators
+            available in DPF Entry.
+
+            +++
+            .. button-link:: ENTRY
+               :color: secondary
+               :expand:
+               :outline:
+               :click-parent:              
+
+   .. grid-item::
+        .. card:: Operators (Premium)
+            :img-top: _static/assets/premium.png
+            :link-type: doc
+            :link: operator_reference_premium
+
+            Click here to get started with operators
+            available in DPF Premium.
+
+            +++
+            .. button-link:: PREMIUM
+               :color: secondary
+               :expand:
+               :outline:
+               :click-parent:
+
+.. note::
+    For Ansys 2023R1 and older, the Premium documentation should be used.
+    Depending on the server version, some operators listed there might
```

Comment:
Depending on the server version, some operators in the documentation may not be available.